packages/babel-plugin-transform-object-rest-spread/test/fixtures/object-rest/variable-destructuring/expected.js

```diff
@@ -0,0 +1,13 @@
+var z = {};
+var x = babelHelpers.objectWithoutProperties(z, []);
+
+var { x1 } = z;
+var y1 = babelHelpers.objectWithoutProperties(z, ["x1"]);
+var { [a]: b } = z;
+var c = babelHelpers.objectWithoutProperties(z, [a]);
```

Comment:
(unrelated but) this breaks in real code when `typeof a` is not string or symbol :(